docs/development.rst

```diff
@@ -537,27 +537,26 @@ Containerization
 ****************
 
 This repo is integrated with Dockerhub and the Docker image will be automatically
-built upon pushes to master on Dockerhub.
+built upon each release on GitHub. Each Docker image is tagged with the corresponding release.
 
-A Docker image can also be locally built with the following steps.
-
-Build image:
+Enter a Docker container from Dockerhub:
 
 .. code-block:: bash
 
-    $ sudo docker build -t tskit/msprime .
+    $ sudo docker run -it tskit/msprime:<release>
```

Comment:
perhaps add an example of what that'd look like, with a particular release number?

Reply:
looks good